modules/graceful-shutdown.adoc

```diff
@@ -1,102 +1,90 @@
+. Mark all the nodes in the cluster as unschedulable. You can do this from your cloud provider's web console, or run the following loop:
 +
-[source,terminal]
-----
-$ oc debug node/<node_name>
-----
-
-... Change your root directory to `/host` by running the following command:
-+
-[source,terminal]
+[source,yaml]
 ----
-sh-4.4# chroot /host
-----
-
-... Check the kubelet client certificate expiration date by running the following command:
-+
-[source,terminal]
-----
-sh-5.1# openssl x509 -in /var/lib/kubelet/pki/kubelet-client-current.pem -noout -enddate
+$ for node in $(oc get nodes -o jsonpath='{.items[*].metadata.name}'); do echo ${node} ; oc adm cordon ${node} ; done
 ----
 +
 .Example output
 [source,terminal]
 ----
 notAfter=Jun  6 10:50:07 2023 GMT
 ----
-
 ... Check the kubelet server certificate expiration date by running the following command:
 +
+----
 [source,terminal]
 ----
 sh-5.1# openssl x509 -in /var/lib/kubelet/pki/kubelet-server-current.pem -noout -enddate
-----
 +
 .Example output
 [source,terminal]
 ----
-notAfter=Jun  6 10:50:07 2023 GMT
+ci-ln-mgdnf4b-72292-n547t-master-0
+node/ci-ln-mgdnf4b-72292-n547t-master-0 cordoned
+ci-ln-mgdnf4b-72292-n547t-master-1
+node/ci-ln-mgdnf4b-72292-n547t-master-1 cordoned
+ci-ln-mgdnf4b-72292-n547t-master-2
+node/ci-ln-mgdnf4b-72292-n547t-master-2 cordoned
+ci-ln-mgdnf4b-72292-n547t-worker-a-s7ntl
+node/ci-ln-mgdnf4b-72292-n547t-worker-a-s7ntl cordoned
+ci-ln-mgdnf4b-72292-n547t-worker-b-cmc9k
+node/ci-ln-mgdnf4b-72292-n547t-worker-b-cmc9k cordoned
+ci-ln-mgdnf4b-72292-n547t-worker-c-vcmtn
+node/ci-ln-mgdnf4b-72292-n547t-worker-c-vcmtn cordoned
 ----
-
-... Exit the debug session.
-
-... Repeat these steps to check certificate expiration dates on all control plane nodes. To ensure that the cluster can restart gracefully, plan to restart it before the earliest certificate expiration date.
-
-. Shut down all of the nodes in the cluster. You can do this from your cloud provider's web console, or run the following loop:
+... Check the kubelet server certificate expiration date by running the following command:
 +
-[source,terminal]
+. Evacuate the pods using the following method:
++
+[source,yaml]
 ----
-$ for node in $(oc get nodes -o jsonpath='{.items[*].metadata.name}'); do oc debug node/${node} -- chroot /host shutdown -h 1; done <1>
+$ for node in $(oc get nodes -l node-role.kubernetes.io/worker -o jsonpath='{.items[*].metadata.name}'); do echo ${node} ; oc adm drain ${node} --delete-emptydir-data --ignore-daemonsets=true --timeout=15s ; done
 ----
-<1> `-h 1` indicates how long, in minutes, this process lasts before the control-plane nodes are shut down. For large-scale clusters with 10 nodes or more, set to 10 minutes or longer to make sure all the compute nodes have time to shut down first.
++
+. Shut down all of the nodes in the cluster. You can do this from your cloud provider’s web console, or run the following loop:
 +
```

Comment:
this step lacked of command of shutting down nodes, like "for node in $(oc get nodes -o jsonpath='{.items[*].metadata.name}'); do oc debug node/${node} -- chroot /host shutdown -h 1; done"